package.json

```diff
@@ -64,6 +64,7 @@
     "@tanstack/react-form": "^1.19.2",
     "@tanstack/react-table": "^8.21.3",
     "@tanstack/react-virtual": "^3.13.12",
+    "clsx": "^2.1.1",
```

Comment:
Can we avoid adding a dependency for just a simple use? In that case you can directly concatenate the classNames.